spec/requests/api/v1/user_request_spec.rb

```diff
@@ -132,11 +132,65 @@
 
 
     get "/api/v1/users/#{user1.id}/find_runner"
-    
+
     expect(response).to be_successful
     resp = JSON.parse(response.body, symbolize_names: true)[:data]
     expect(resp.count).to eq(3)
     expect(resp.first[:attributes].count).to eq(8)
     expect(resp.include?(user5)).to eq(false)
   end
+
+  it "A user can be created and logged in at the same time" do
```

Comment:
Add test to check to see if a user can be created and logged in at the same time.